src/public/services/__tests__/PublicFormAuthService.test.ts

```diff
@@ -0,0 +1,116 @@
+import axios from 'axios'
+import { ObjectId } from 'bson'
+import { mocked } from 'ts-jest/utils'
+
+import * as PublicFormAuthService from '../PublicFormAuthService'
+
+jest.mock('axios')
+const MockAxios = mocked(axios, true)
+
+const MOCK_REDIRECT_URL = 'redirectURL'
+const MOCK_FORM_ID = new ObjectId().toHexString()
+
+describe('PublicFormAuthService', () => {
+  afterEach(() => jest.resetAllMocks())
+  describe('createRedirectURL', () => {
```

Comment:
I think this test suite is left over from rebasing

Reply:
yeap! this is because the tests PR isn't in yet due to an error from my end. idt this will show up when it is merged into `develop`!